spikeinterface/postprocessing/principal_component.py

```diff
@@ -373,6 +373,7 @@ def _fit_by_channel_local(self, sparsity, n_jobs, progress_bar):
                 with open(pca_model_file, 'rb') as fid:
                     pca_models.append(pickle.load(fid))
                 pca_model_file.unlink()
+            shutil.rmtree(tmp_folder)
 
```

Comment:
what is this ?

Reply:
A temporary folder needed to parallelize training